### PR TITLE
JA-02 Create unit tests for UserService

### PR DESCRIPTION
**Pull request by tatianaburii**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation 'app.just.common:common-app:1.6-SNAPSHOT'
     implementation 'org.postgresql:postgresql'
     implementation 'org.flywaydb:flyway-core'
+    implementation 'org.projectlombok:lombok:1.18.22'
```
Comment (TeraDen) [inline build.gradle:34]: Lombok already exist

File: src/test/java/service/UserServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  @Mock
+  UserRepository userRepository;
+  @Mock
+  UserEntityMapper userEntityMapper;
+  @InjectMocks
+  UserService userService;
+
+  @Test
+  void findAll() {
```
Comment (TeraDen) [inline src/test/java/service/UserServiceTest.java:35]: Use informative name 
as example, template given_when_then

File: src/test/java/service/UserServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@FieldDefaults(level = AccessLevel.PACKAGE)
+public class UserServiceTest {
+
+  @Mock
+  UserRepository userRepository;
+  @Mock
```
Comment (TeraDen) [inline src/test/java/service/UserServiceTest.java:29]: As an option, you can use mode with call real method

File: src/test/java/service/UserServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@FieldDefaults(level = AccessLevel.PACKAGE)
```
Comment (TeraDen) [inline src/test/java/service/UserServiceTest.java:24]: Why do you need it?

Comment (tatianaburii) [reply src/test/java/service/UserServiceTest.java]: What do you ask about? @ExtendWith(MockitoExtension.class) or FieldDefaults(level = AccessLevel.PACKAGE)?

**Pull request merged**